src/ui/components/Button/index.js

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import './Button.scss';
+
+export default class Button extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    href: PropTypes.string,
+  }
+
+  render() {
+    const { children, className, href, ...rest } = this.props;
```

Comment:
Since it gets used in the `if` I thought it would be good to pull it out explicitly.